techniques/general/ensuring-single-pointer.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Providing controls to achieve the same result as path based or multipoint gestures</title>
+		<link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
+	</head>
+	<body>
+		<h1>Providing controls to achieve the same result as path based or multipoint gestures</h1>
+		<section id="meta">
+			<h2>Metadata</h2>
+			<p id="id"></p>
+			<p id="technology">general</p>
+			<p id="type">sufficient</p>
+		</section>
+		<section id="applicability">
+			<h2>When to use</h2>
+			<h3>Applicability</h3>
+			<p>Any technology that supports pointer input (e.g. supporting any or all of the following: mouse pointer, touch on touch screen or trackpad, stylus input, or laser pointer input).</p>
+		</section>
+		<section id="description">
+			<h2>Description</h2>		
+			<p>The objective of this technique is to ensure that users who have difficulties performing path-based gestures can operate a content slider with a single pointer (e.g., a single tap on a touch screen or a single mouseclick). A content slider contains chunks of content in a row. Usually several chunks of content are hidden, only one chunk is visible at any time. Left and right horizontal swiping over the visible part of the slider brings adjacent hidden chunks of content into view. Providing controls (for example, arrow buttons) as alternative means of input allows advancing the slider with single pointer input.</p>
```

Comment:
"mouseclick" > "mouse click" ?

Reply:
"are hidden, only one chunk" > "are hidden, and only one chunk"

Reply:
Done